src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+   public String getRandomBall() {
+       Random random = new Random();
+       int number = random.nextInt(100);
+       return "The ball with color:" + new ColorSupplier().getRandomColor() + " and number:" + number;
```

Comment:
same lets make Random and ColorSupplier as class level variables, not to create new instances every time we call getRandomBall() method